src/events/broker.py

```diff
@@ -167,20 +168,13 @@ def _on_start(self, event: StartEvent | PebbleReadyEvent) -> None:  # noqa: C901
         if not self.upgrade.idle:
             return
 
-        if self.charm.state.kraft_mode:
-            self._init_kraft_mode()
-
         # FIXME ready to start probably needs to account for credentials being created beforehand
```

Comment:
I think this comment can be removed. Credentials issues on the start process were solved already